torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -1023,17 +1023,12 @@ const std::vector<std::string> functions = {
                        eps : float,
                        cudnn_enabled : bool):
 
-            output, save1, save2, reserve, impl_idx = torch._batch_norm_impl_index(
-                input, weight, bias, running_mean, running_var, training,
-                momentum, eps, cudnn_enabled)
-            has_weight = weight is not None
-            has_bias = bias is not None
+            output, mean, rstd = torch.native_batch_norm(input, weight, bias, running_mean, running_var, training, momentum, eps)
```

Comment:
I think for forward, we can almost always safely fuse it (and mark our backend as native_batch_norm).
But for backwards, we would want to opt out of fusion if forward is running with cudnn, simply because of the opaque data structure that we can't consume.

Reply:
The other versions of batch norm don't return `mean and rstd`, which we need for batch norm backward.

Reply:
Doesn't `_batch_norm_impl_index` return all the things that we want (and even things that we don't care -> workspace :) )